rest/doc_api.go

```diff
@@ -209,10 +209,37 @@ func (h *handler) handleGetAttachment() error {
 	}
 	h.setHeader("Content-Length", strconv.FormatUint(uint64(len(data)), 10))
 
+	// #720
+	setContentDisposition := h.privs == adminPrivs
+
 	h.setHeader("Etag", strconv.Quote(digest))
-	if contentType, ok := meta["content_type"].(string); ok {
+
+	// Request will be returned with the same content type as is set on the attachment. The caveat to this is if the
+	// attachment has a content type which is vulnerable to a phishing attack. If this is the case we will return with
+	// the Content Disposition header so that browsers will download the attachment rather than attempt to render it
+	// unless overridden by config option. CBG-1004
+	contentType, contentTypeSet := meta["content_type"].(string)
+	if contentTypeSet {
 		h.setHeader("Content-Type", contentType)
 	}
+
+	if !h.db.ServeInsecureAttachmentTypes {
+
+		if contentTypeSet {
+			contentTypeFirst := strings.Split(contentType, ";")[0]
```

Comment:
As discussed earlier, add comment to explain this.

Reply:
Ah yeah woops